zone/client.cpp

```diff
@@ -12258,3 +12258,34 @@ int Client::GetEXPPercentage()
 
 	return static_cast<int>(std::round(scaled * 100.0 / 330.0)); // unscaled pct
 }
+
+std::vector<Mob*> Client::GetRaidOrGroupOrSelf(bool clients_only)
+{
+	std::vector<Mob*> v;
+
+	if (IsRaidGrouped()) {
+		Raid* r = GetRaid();
+
+		if (r) {
+			for (const auto& m : r->members) {
+				if (m.member && (!clients_only || !m.is_bot)) {
+					v.emplace_back(m.member);
+				}
+			}
+		}
+	} else if (IsGrouped()) {
+		Group* g = GetGroup();
+
+		if (g) {
+			for (const auto& m : g->members) {
+				if (m && (!clients_only || !m->IsBot())) {
```

Comment:
Can mercs also be in groups on emu? Maybe these should just check for `|| IsClient()` instead of !bot

Reply:
> Can mercs also be in groups on emu? Maybe these should just check for `|| IsClient()` instead of !bot

Ah yeah. Forgot about Mercenaries.